t/configuration-loading-with-oidc.t

```diff
@@ -0,0 +1,232 @@
+use lib 't';
+use Test::APIcast 'no_plan';
```

Comment:
Can't it be a blackbox test?

Reply:
I based these tests on the ones we already have that test the configuration loading.
I'm not sure if a Blackbox test would work in this case because we'd need to define all those locations to get the configuration and there wouldn't be a "---configuration" directive.

Reply:
It would work with lazy loading the configuration. 

My point is that Blackbox is 100% real APIcast. This is somewhat similar to APIcast but not really. And we really should aim to not add more of these tests as they are not compatible with the liquid templating of the nginx config.